samples/web/content/apprtc/constants.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python2.4
+#
+# Copyright 2014 Google Inc. All Rights Reserved.
+
+"""AppRTC Constants
+
+This module contains the constants used in AppRTC Python modules.
+"""
+
+LOOPBACK_CLIENT_ID = 'LOOPBACK_CLIENT_ID'
+
+TURN_BASE_URL = 'https://computeengineondemand.appspot.com'
+TURN_URL_TEMPLATE = '%s/turn?username=%s&key=%s'
+CEOD_KEY = '4080218913'
+
+WSS_HOST = 'apprtc-ws.webrtc.org'
```

Comment:
remind me, why can't we just store this as a single constant WSS_URL (and have a single param to overrride it)?